app/styles/local.css

```
@@ -1,593 +1,593 @@
+.form-group.has-success.has-feedback .form-text
+{
+  color: #2ecc71;
+}
+
+.form-group.has-success.has-feedback .ember-basic-dropdown-trigger
+{
+  color: #2ecc71;
+  border-color: #2ecc71;
+}
+
+lock {
```

Biome (2.4.7)

[error] Unknown type selector is not allowed.

(lint/correctness/noUnknownTypeSelector)